### ISSUE TITLE
Validation of Protocol Buffer

### ISSUE THREAD

**Issue by tusharmakkar08** [alecthomas/voluptuous#222]
Hey @alecthomas 

What are your views on extending Voluptuous for validating [protocol buffers](https://developers.google.com/protocol-buffers/) as well? Something similar to [this](https://github.com/gobengo/protobuf-validator).

Thanks

**Comment by alecthomas** [comment]
What form are you imagining the support taking? Off the top of my head, translating the protobuf to a Python dict, then passing that to voluptuous, seems like a straightforward change. Voluptuous could optionally import some module that knows how to translate protobuf objects to dicts perhaps.

**Comment by tusharmakkar08** [comment]
Yeah I was also thinking the same. Maybe we can use [protobuf-to-dict](https://github.com/benhodgson/protobuf-to-dict). What is your opinion about the [mentioned](https://pypi.python.org/pypi/protobuf-to-dict) package? BTW there is another [similar package](https://pypi.python.org/pypi/protobuf3-to-dict/) which supports protobuf-3.  It is fork of the old project.
